framework/utils/InputData.py

```diff
@@ -556,7 +556,7 @@ def generateLatex(cls, recDepth=0):
       @ In, recDepth, int, optional, recursion depth of printing
       @ Out, msg, str, LaTeX string representation of user manual entry
     """
-    name = cls.name
+    name = cls.name#name = re.sub(r'(?<!\\)_', r'\_', cls.name)
```

Comment:
Um, this should be:
```name = re.sub(r'(?<!\\)_', r'\_', cls.name)```

Reply:
(As in remove name = cls.name, and replace with what is after the comment.)

Reply:
Josh, I did that at the beginning but tried a breakpoint there and every time it hits the breakpoint `cls.name` is exactly similar to the `name` after executing this line. Probably because we do not have names with underscores!! Anyway, I pushed it back again.

Reply:
Do we actually have a standard that no underscores are used in the input file? I don't think that coding standard is extended to the input.

Reply:
No, we don't, that's why we are escaping in the names of variables.